### ISSUE TITLE
:unprintable characters not scrubbed from script tag

### ISSUE THREAD

**Issue by bbugh** [flavorjones/loofah#124]
The `unprintable` scrubber is not removing unprintables from a script tag. Examples:

```ruby
def strip_unprintables(raw_content)
  Loofah.fragment(raw_content).scrub!(:unprintable).to_s
end  

# bug: regular tag + script
> strip_unprintables("hack<script>\u2028</script>protected<div>\u2028</div>\u2028")
=> "hack<script>\u2028</script>protected<div></div>"
                ^^^^^^

# bug: forbidden tag + script
> strip_unprintables("hack<script>\u2028</script>protected<div>\u2028</div>\u2028<object>\u2028</object>")
=> "hack<script>\u2028</script>protected<div></div><object></object>"
                ^^^^^^

# bug: other forbidden tag + script
> strip_unprintables("hack<script>\u2028</script>protected<div>\u2028</div>\u2028<iframe>\u2028</iframe>")
=> "hack<script>\u2028</script>protected<div></div><iframe></iframe>"
                ^^^^^^

# bug: when using `strip`
> raw_content = "hack<script>\u2028</script>protected<div>\u2028</div>\u2028<iframe>\u2028</iframe>"
> Loofah.fragment(raw_content).scrub!(:unprintable).scrub!(:strip).to_s
=> "hack\u2028protected<div></div>"
        ^^^^^^

> Loofah.fragment(raw_content).scrub!(:strip).scrub!(:unprintable).to_s
=> "hack\u2028protected<div></div>"
        ^^^^^^

# correct: but `prune` works (in either order)
Loofah.fragment(raw_content).scrub!(:prune).scrub!(:unprintable).to_s
=> "hackprotected<div></div>"
```

**Comment by flavorjones** [comment]
Great catch. We're not looking at CDATA nodes, which is what a `<script>` tag contains. Will fix.

**Comment by flavorjones** [comment]
Committed a fix. Will be in the next release.

**Comment by flavorjones** [comment]
This is fixed in v2.1.0 that shipped earlier today.
